src/openfermion/utils/_sparse_tools.py

```diff
@@ -69,7 +69,7 @@ def jordan_wigner_ladder_sparse(n_qubits, tensor_factor, ladder_type):
         n_qubits(int): Number qubits in the system Hilbert space.
 
     Returns:
-        The corresponding SparseOperator.
+        The corresponding Scipy sparse matrix.
```

Comment:
Very good to catch these, looks like the rest of us have missed it for a long time :P